common/src/main/java/com/wynntils/gui/screens/overlays/OverlayManagementScreen.java

```diff
@@ -68,16 +68,15 @@ public class OverlayManagementScreen extends Screen {
             Component.literal("Shift-Middle click on an overlay to reset it to it's original state.")
                     .withStyle(ChatFormatting.RED));
 
-    private static final List<Component> CLOSE_TOOLTIP_LINES =
-            List.of(Component.literal("Click here to stop editing and reset changes."));
+    private static final Component CLOSE_TOOLTIP_LINES =
+            Component.literal("Click here to stop editing and reset changes.");
 
-    private static final List<Component> TEST_TOOLTIP_LINES = List.of(
-            Component.literal("Click here to toggle test mode."),
-            Component.literal("In test mode, you can see how your overlay setup would look in-game,"),
-            Component.literal("using preview render mode."));
+    private static final Component TEST_TOOLTIP_LINES = Component.literal("Click here to toggle test mode.")
```

Comment:
Will these tooltip still appear with one message per line, or will the append just join them together in a huge string, that is then wrapped "at will" by the tooltip renderer?

Reply:
I just found another case of MC source code being incredibly dumb. Why would you create a `renderWithTooltp` only to not call `super.render` but `this.render`...? Anyways, I am working on this.

Reply:
I fixed it. It looks okay, MC wraps it by default.